packages/autorest.go/src/generator/options.ts

```diff
@@ -3,12 +3,13 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { capitalize, comment } from '@azure-tools/codegen';
 import { Session } from '@autorest/extension-base';
-import { CodeModel, GroupProperty } from '@autorest/codemodel';
+import { CodeModel, ConstantSchema, GroupProperty, ImplementationLocation, Parameter, SchemaType } from '@autorest/codemodel';
 import { values } from '@azure-tools/linq';
-import { contentPreamble, sortAscending } from './helpers';
+import { commentLength } from '../common/helpers';
+import { contentPreamble, hasDescription, sortAscending } from './helpers';
```

Comment:
Could helpers in generator be part of common/helpers?

Reply:
The way this is currently architected, `./helpers.ts` contains shared code across contents in the `generator` directory, and `common/helpers` contains code shared across code in the `generator` and `transform` directories.  Note that `common/helpers` will be going away with the adapter refactor as there will no longer be common code across those directories.